Library/Homebrew/cmd/update-report.rb

```diff
@@ -136,6 +140,21 @@ def update_report
     Commands.rebuild_commands_completion_list
     link_completions_manpages_and_docs
     Tap.each(&:link_completions_and_manpages)
+
+    return if new_repository_version.blank?
+
+    ohai "Homebrew was updated to version #{new_repository_version}"
+    puts <<~EOS
+      The changelog can be found at:
+        #{Formatter.url("https://github.com/Homebrew/brew/releases/tag/#{new_repository_version}")}
```

Comment:
Might want to link to the blog instead rather than as well for major/minor releases?

Reply:
Makes sense to me. 👍